utils/check_audio_consistency.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I feel it might be better if we call it check_audio_md5.sh or check_audio_checksum.sh